frontend/src/containers/RecipePage/Recipe.css

```diff
@@ -0,0 +1,177 @@
+.recipe-overall-container {
+  margin: 0px auto;
+  padding: 0px;
+}
+
+.cancelButton, .editButton {
+  display: inline-block;
+  float: right;
+  margin: 20px 20px;
+  border: 2px solid #ffa616;
+  margin-top: 70px;
+  color: #ffa616;
+}
+
+.recipe-difficulty {
+  padding-right: 5px;
+  color: #4EC8ED;
+}
+
+.recipe-cancel-button, .recipe-edit-button {
+  display: flex;
+  float: right;
+  margin: 20px 20px;
+  border: 2px solid #ffa616;
+  border-radius: 20%;
+  color: #ffa616;
+  font-family: 'Nunito', sans-serif;
+  font-size: 18px;
+  height: 3.5em;
+  padding-left: .5em;
+  padding-right: .8em;
+  line-height: 3em;
+  font-weight: bold;
+  width: 120px;
+  text-align: center;
+  margin-top: 70px;
+}
+
+#delete-icon-text, #edit-icon-text {
+  display: inline;
+  margin-left: 6px;
+}
+
+.recipe-icon {
+  margin-top: 50px;
+  height: 80px;
+  line-height: 60px;
+  text-align: left;
+  font-size: 23.4px;
+  color: white;
+  padding: 0px;
+}
+
+.recipe-photo {
+  width: 90%;
+  height: 500px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.tags-bar {
+  display: flex;
+  justify-content: center;
+}
+
+#recipe-descrip {
+  text-align: center;
+  padding: 1% 3% 3% 3%;
+  color: #696969;
+  font-size: 20px;
+}
+
+#hr {
+  border: 0.5px solid lightgrey;
+  width: 90%;
+  margin: 0 auto;
+}
+
+#ingredient-box {
+  display: flex;
+  justify-content: center;
+  margin-top: 1%;
+  padding: 1% 3% 3% 3%;
+  margin-bottom: -1%;
+}
+
+#ingredient-ul {
+  column-count: 2;
+  column-gap: 50px;
+  padding: 6% 3% 3% 3%;
+  width: 100%;
+  color: #696969;
+  font-size: 20px;
+}
+
+#recipe-title, #direction-title, #ingredient-title {
+  margin-top: 2%;
+  text-align: center;
+  padding: 1% 3% 3% 3%;
+  font-size: 50px;
+  font-weight: bold;
+}
+
+#direction-container {
+  margin-bottom: 20px;
+  text-align: left;
+  padding: 1% 3% 3% 3%;
+}
+
+.single-direction {
+  text-align: left;
+  margin: 30px 0;
+}
+
+.direction-index {
+  display: inline;
+  margin-right: 8px;
+  color: #696969;
+  font-size: 32px;
+}
+
+.direction-action {
+  display: inline;
+  vertical-align: top;
+  font-size: 24px;
```

Comment:
Can we make the directions use the same font size as the snippet and ingredients? Maybe that means bumping those up to 24px, or dropping this down to 20px, whichever way looks better.

Reply:
Changed it to 20px for all. Let me know if you think 24px would be better, otherwise we should be good.

Reply:
I changed it to put a bit bigger to match design doc